cgmanifest.json

```diff
@@ -4094,76 +3784,6 @@
         }
       }
     },
-    {
-      "component": {
-        "type": "other",
-        "other": {
-          "name": "libXmu",
-          "version": "1.1.3",
-          "downloadUrl": "https://www.x.org/pub/individual/lib/libXmu-1.1.3.tar.bz2"
-        }
-      }
-    },
-    {
-      "component": {
-        "type": "other",
-        "other": {
-          "name": "libXpm",
-          "version": "3.5.13",
-          "downloadUrl": "https://www.x.org/pub/individual/lib/libXpm-3.5.13.tar.bz2"
-        }
-      }
-    },
-    {
-      "component": {
-        "type": "other",
-        "other": {
-          "name": "libXrandr",
```

Comment:
Why are we removing packages like these from the `cgmanifest.json`? I think we still have them. And is this related to the `openjdk` update?

Reply:
A number of the x.org dependencies were added to the toolchain in order to compile jdk8. Since that's being removed, so are the dependencies. Although it looks like some of these have been added as SPECS from the coreui repo. I'll double check and restore those case-by-case.